tests/components/dlna_dmr/conftest.py

```diff
@@ -129,6 +129,7 @@ def dmr_device_mock(domain_data_mock: Mock) -> Iterable[Mock]:
         device.manufacturer = "device_manufacturer"
         device.model_name = "device_model_name"
         device.name = "device_name"
+        device.preset_names = ["preset1", "preset2"]
```

Comment:
This is written to entity registry, if not set here we try to write a `Mock` which is caught during test shutdown